CONTRIBUTING.md

```diff
@@ -147,29 +147,61 @@ from the main (upstream) repository:
     git pull --ff upstream master
     ```
     
-## <a name="building-docker"></a>Building a Docker image
-In order to build a Docker image of IBeam, you need to first ensure both Chrome Driver and CPW Gateway are available in `./copy_cache` directory under:
+## <a name="building-docker"></a>Building a Docker Image
 
-* `./copy_cache/chrome_driver` - for the Chrome Driver
-* `./copy_cache/clientportal.gw` - for the Gateway
+The following commands can be used to build the IBeam image after navigating to the root directory of the repository.
 
-You can automate this process by carrying out the following:
+### <a name="local-builds"></a>Local Single-Platform Builds
 
-* Navigate to the root folder of IBeam in your console
-* Run `pip install -r dev-requirements.txt` to install [`invoke` package][invoke]
-* Define the following environment variable paths, contents of which will be copied to the image
-    * `IBEAM_CHROME_DRIVER_PATH` env var pointing at the Chrome Driver directory to be copied from
-    * `IBEAM_GATEWAY_DIR` env var pointing at the Gateway directory to be copied from
-* Make sure `invoke.exe` is in your `$PATH`
-* Run `invoke.exe copyClientportal` and `invoke.exe copyChromeDriver`
-* (optionally) Ensure both `copy_cache/chrome_driver` and `copy_cache/clientportal.gw` are present
+To build the IBeam image for local testing, run:
 
-Finally, run: 
-
-```posh
+```shell
 docker build -t ibeam .
 ```
 
+Alternatively, `docker-compose` can be used to build and run a local IBeam instance using variables in `env.list`:
+
+```shell
+docker-compose up -d --build
+```
+
+### <a name="multi-platform-builds"></a>Multi-Platform Builds
+
+The commands below can be used to setup `docker buildx` for multi-platform IBeam builds supporting `amd64` and `arm64` machines. These commands only need to be executed once per machine and are not required for subsequent multi-platform builds.
```

Comment:
Great you add this little introduction, I'm thinking maybe we could even add small subheadings like 

### <a name="multi-platform-builds"></a>Multi-Platform Builds
#### Before you start
The commands below can be used...


#### Build
The next two commands can now be used to build...

Reply:
I have added in your recommended subheadings and will push the changes in the next commit.